dask/array/tests/test_testing.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import numpy as np
+
+import dask.array as da
+from dask.array.utils import assert_eq
+
+
+def test_assert_eq_checks_scalars():
+    # https://github.com/dask/dask/issues/2680
+    a = da.from_array(np.array([0]), 1)[0]
+    b = np.array([1])[0]
+    with pytest.raises(AssertionError):
+        assert_eq(a, b)
+
+
+def test_assert_eq_empty_passes():
+    # Treat empty arrays as equivalent
+    assert_eq(np.array([[], []]), np.array([]))
```

Comment:
I *think* this was the intention of that code block original, based on the comment `# Treat all empty arrays as equivalent`. Does that sound right to you @mrocklin?